### DISCUSSION TITLE
Gap

### DISCUSSION THREAD

**Discussion by ljungis** [supermerill/SuperSlicer#3623] [Q&A]
![cap](https://user-images.githubusercontent.com/37312627/216598727-db562986-f675-46b8-944b-28991bc69b0f.png)
[cap.zip](https://github.com/supermerill/SuperSlicer/files/10578632/cap.zip)

Hi, as seen in the attached picture there is gap in the infill/perimeter of the first layer.
How can I avoid that.

**Comment by rfnovo** [comment]
Seems like your nozzle is too high and your bed is too cold. Try adjust your Z Offset and set bed temperature between 70 to 80.

**Reply by ljungis** [reply]
Thank you rfnovo. I set the Z offset to -0,1 mm and raised the bed temperature.
Now the part looks much better.
